vividus-allure-adaptor/src/main/resources/org/vividus/report/allure/spring.xml

```diff
@@ -51,6 +51,8 @@
 
     <bean id="allurePluginsProvider" class="org.vividus.report.allure.AllurePluginsProvider" />
 
+    <bean class="org.vividus.report.allure.notification.NotificationsSender" />
```

Comment:
We can move it to the generator definition since we don't use it outside generator